R/mbbs.R

```diff
@@ -130,6 +130,37 @@ create_route_level_counts_0 <- function(ebird, stop_level_data, taxonomy, config
         })()
     })()
 
+  # Compare stop_level and ebird counts
+  dplyr::left_join(
```

Comment:
@IJBG - please check this logic carefully. I'm not finding any differences, but maybe they were all fixed?

Reply:
rcount is getting lost in the left_join
![image](https://github.com/user-attachments/assets/5b16b5ea-c0bf-4e22-a226-46cb3687038a)

Reply:
HM. Getting the same problem left_joining the other way, with ebird as the base and then the stop to route. Is something happening here where they're like, not overlapping? Maybe so because when I filter the two to both eg. Canada Goose and then anti-join, no overlaps at all. Same if I anti_join the whole datasets. Doesn't make sense to me however that there would be no overlaps between these two dfs.

I'm going to go take a look at how I handled finding these mismatches in prev. code

Reply:
OH. Line 124 you're removing all the ebird records that are already in stop-level data.